package.json

```diff
@@ -26,7 +25,7 @@
     "walk-sync": "^0.3.3"
   },
   "devDependencies": {
-    "broccoli": "^2.0.1",
+    "broccoli": "broccolijs/broccoli#master",
```

Comment:
should we wait for release?

Reply:
Yup. I want to cut w release soon with the BROCCOLI_ENV stuff in it.

Reply:
V2.1.0